pkgs/development/python-modules/mailsuite/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, isPy3k, pythonPackages }:
+
+buildPythonPackage rec {
+  pname = "mailsuite";
+  version = "1.6.1";
+
+  disabled = !isPy3k;
```

Comment:
```suggestion
```

python 2 packages are extra.

Reply:
What do you mean? Our `python` package is still Python 2 and `pythonPackages` are built with it.

Reply:
pythonPackages is not shared with python3Packages anymore. They are defined in two differently files and if you do not include an entry in python2-packages than the package is not available under pythonPackages.

Reply:
That doesn't seem to be correct.
```
$ nix-build . -A pythonPackages.parsedmarc
error: parsedmarc-6.12.0 not supported for interpreter python2.7
```
Also, the comment at the top of `python2-packages.nix` says that they're overlayed on top of the Python 3 packages:
```
# Extension with Python 2 packages that is overlayed on top
# of the Python 3 packages set. This way, Python 2+3 compatible
# packages can still be used.
```